shared/AppInsightsCommon/src/Interfaces/IPageViewTelemetry.ts

```diff
@@ -31,6 +31,17 @@ export interface IPageViewTelemetry extends IPartC {
      * isLoggedIn - boolean is user logged in
      */
     isLoggedIn?: boolean;
+
+    /**
+     * Property bag to contain additional custom properties (Part C)
+     */
+    properties?: {
+        /**
+         * The number of milliseconds it took to load the page. Defaults to undefined. If set to default value, page load time is calculated internally.
+         */
+        duration?: number;
```

Comment:
Is duration available to specify in PageViewPerformance events as well?, I can see is internal currently

Reply:
I'm not sure if the duration value from pageviewperformance ends up on the end-to-end trace gantt chart. AFAIK it does not and it uses the pageview duration instead.